eo-maven-plugin/src/test/java/org/eolang/maven/tojos/ForeignTojosTest.java

```diff
@@ -116,8 +120,67 @@ void findsAnyTojoIfSeveralTojosWithTheSameIdWereAdded() {
         );
     }
 
+    @ParameterizedTest
+    @MethodSource("tojoFunctions")
+    void throwsExceptionIfKeyWasNotFoundInTojo(
+        final String key,
+        final Func<ForeignTojo, Object> method) {
+        final ForeignTojo tojo = this.tojos.add("string");
+        Assertions.assertThrows(
+            IllegalStateException.class,
+            () -> method.apply(tojo),
+            String.format("Should throw an exception if key='%s' was not found in Tojo", key)
+        );
+        final IllegalStateException thrown = Assertions.assertThrows(
```

Comment:
@Yanich96 this assertion and the next one should be moved to the separated test.